Formula/nifi-registry.rb

```diff
@@ -5,19 +5,24 @@ class NifiRegistry < Formula
   mirror "https://archive.apache.org/dist/nifi/nifi-registry/nifi-registry-0.8.0/nifi-registry-087.0-bin.tar.gz"
   sha256 "db9f390eb4e1f99fb861c437be95729dfcb203f49de4b9cb3130a612a4ae288d"
   license "Apache-2.0"
+  revision 1
 
   livecheck do
     url :stable
   end
 
   bottle :unneeded
 
+  depends_on "openjdk"
+
   def install
     libexec.install Dir["*"]
     rm Dir[libexec/"bin/*.bat"]
 
     bin.install libexec/"bin/nifi-registry.sh" => "nifi-registry"
-    bin.env_script_all_files libexec/"bin/", NIFI_REGISTRY_HOME: libexec
+    bin.env_script_all_files libexec/"bin/",
+                             NIFI_REGISTRY_HOME: libexec,
+                             JAVA_HOME:          Formula["openjdk"].opt_prefix
```

Comment:
I think this should be overwrittable

Reply:
What do you mean by that?

Reply:
Something like https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/jboss-forge.rb#L15